gallery/plot_visualization_utils.py

```diff
@@ -418,10 +407,23 @@ def show(imgs):
 # The coco keypoints for a person are ordered and represent the following list.\
 
 coco_keypoints = [
-    "nose", "left_eye", "right_eye", "left_ear", "right_ear",
-    "left_shoulder", "right_shoulder", "left_elbow", "right_elbow",
-    "left_wrist", "right_wrist", "left_hip", "right_hip",
-    "left_knee", "right_knee", "left_ankle", "right_ankle",
+    "nose",
```

Comment:
Same as above. So the question basically becomes: should we keep the minimal changes that align the galleries with the rest of our code or shall I revert every non-grammar-or-typo change here?